### PR TITLE
Database branch

### PR DESCRIPTION
**Pull request by benfrey**
Will have the expressAPIServer and expressWebServerExample directories working properly (the same as the expressWebServerExampleStandalone) by tomorrow night.

### CODE REVIEW

File: expressAPIServer/app.js [hunk 1]
```diff
@@ -1,48 +1,43 @@
...
-app.use(function(req, res, next) {
-  next(createError(404));
+// Connection to the database team Azure DB
+// Eventually we would like to replace this authentificaiton with tokens
+var config =
+{
+    host: 'classy-schedule-database.mysql.database.azure.com';,
+    user: 'db_test';,
+    password: 'fA!6#_&eaU9-EaeJ';,
+    database: 'db_dev';,
```
Comment (gabbiebolcer) [inline expressAPIServer/app.js]: should there be `;` here? when I copied this into my IDE it was giving me errors

**Review by gabbiebolcer — APPROVED** [review]

File: expressAPIServer/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+1. $cd expressAPIServer
+2. $npm install
+3. $npm start
+
+### Connection via cURL
+From another terminal, submit HTTP requests to "http://localhost:4000". Here are some examples to connect to the API via [cURL](https://curl.se/):
+- **View something** (i.e. GET): $curl -X GET "http://localhost:4000/getDept"
+- **View something** (i.e. GET): $curl -X GET "http://localhost:4000/getClass?dept_code=STAT"
+- **Add something** (i.e. POST): $curl -X POST "http://localhost:4000/addClass?dept_code=STAT&class_num=101&class_name=Introduction+to+Statistics"
+    - Note: If an error from database is the response, the entry probably already exists or something more serious happened. Either way, we are working to increase error verbosity.
```
Comment (joelambrecht) [inline expressAPIServer/README.md:22]: Not necessarily increase verbosity- don't want to decrease security.

File: expressAPIServer/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+</br>
+Note: Before deploying to Azure, make sure to change the 
+</br></br>
+Note: We followed this guide for Node.js application deployment to Azure: https://docs.microsoft.com/en-us/azure/app-service/quickstart-nodejs?tabs=linux&pivots=development-environment-azure-portal
+
+### Connection via cURL
+From another terminal, submit HTTP requests to "http://classy-api.ddns.net". Here are some examples to connect to the API via [cURL](https://curl.se/):
+- **View something** (i.e. GET): $curl -X GET "http://classy-api.ddns.net/getDept"
+- **View something** (i.e. GET): $curl -X GET "http://classy-api.ddns.net/getClass?dept_code=STAT"
+- **Add something** (i.e. POST): $curl -X POST "http://classy-api.ddns.net/addClass?dept_code=STAT&class_num=101&class_name=Introduction+to+Statistics"
```
Comment (joelambrecht) [inline expressAPIServer/README.md:44]: add uses [data], so this should be $curl -X POST -d "dept_code=STAT&class_num=220&class_name=Statistics 1" ""http://classy-api.ddns.net/addClass"

Comment (joelambrecht) [reply expressAPIServer/README.md:44]: curl also defaults to get, so you don't need the -X for those commands

File: expressAPIServer/app.js [hunk 4]
```diff
@@ -1,115 +1,92 @@
...
-      res.status(500).send("Error querying database");
+        // Not successful, send status 500, error
+        res.status(500).send("Error querying database");
     });
-
-    res.redirect('/classes:id')
-});
+}
 
 module.exports = app;
```
Comment (joelambrecht) [inline expressAPIServer/app.js:92]: We'll need to compare my API file to yours here- hard to tell what changes are mandatory and which are aesthetic.

**Review by joelambrecht — COMMENTED** [review]
As this gets closer to being shared with other teams, keep in mind what schema you have it pointing to. db_dev is our team's development environment, but we want to direct other teams to cs_dev.

**Pull request closed without merging**